TitaniumTask/app/src/main/java/com/sa/testtask/screens/MainActivity.java

```diff
@@ -61,7 +58,9 @@
 
     @OnTextChanged(R.id.edit_label)
     void onEditTextClick(CharSequence s, int start, int before, int count) {
-        if (s.length() != count) save.setVisibility(View.VISIBLE);
+        //it is not working
```

Comment:
Really? It should work. If you add or delete one character, button save will become visible.

Reply:
Tested on nexus 5x with android 8, problem was that count == s.length all the time.
Only when i pressed enter on IME it helped.

Reply:
Interesting)) I also tested on 5x android 8 (real device) and also on htc with android 5 and it was working. But ok I will try to do somehow another